versions/src/main/resources/fabric.mod.json

```diff
@@ -21,6 +21,7 @@
     "preLaunch": [
       "cc.polyfrost.oneconfig.internal.plugin.OneConfigPreLaunch"
     ],
+    "main": [ "cc.polyfrost.oneconfig.test.TestMod_Test" ],
```

Comment:
can you somehow remove this when its actually built / published

Reply:
will try to